src/main/frontend/app/HeaderContent.scss

```diff
@@ -0,0 +1,140 @@
+
+
```

Comment:
please remove extra lines. You've got a bunch in this file, please review and remove.